lib/theme_check/template.rb

```diff
@@ -58,6 +58,15 @@ def root
       parse.root
     end
 
+    def update!(line, value)
+      lines[line - 1] = value
+      @source = lines.join("\n")
+    end
+
+    def write
+      File.write(path, source)
```

Comment:
I think we should track when files are changed (maybe a ivar?) and not re-write the file when it hasn't change.

This way, it won't touch the files needlessly.

Reply:
ah that's a good point